### DISCUSSION TITLE
In kong Hybrid deployment mode, if dp, What to do if there is insufficient memory

### DISCUSSION THREAD

**Discussion by yuanst-627** [Kong/kong#11728] [Help]
We know that dp will regularly pull configuration information from cp, such as keyauth_credentials
However, if there are too many keyauths and dp has insufficient memory, is there possible problem in running the program at this time?

**Comment by StarlightIbuki** [comment]
OOM is always an unrecoverable error and Kong relies on dynamic memory allocation to work. The only thing we can do is to avoid OOM and no countermeasure is possible.
